packages/cli/test/util/validate-plan.test.ts

```diff
@@ -0,0 +1,118 @@
+import test from 'ava';
+import { createMockLogger } from '@openfn/logger';
+import type { ExecutionPlan } from '@openfn/lexicon';
+import validate from '../../src/util/validate-plan';
+
+const logger = createMockLogger('', { level: 'debug' });
+
+test.afterEach(() => {
+    logger._reset();
+})
+
+test('throws for missing workflow', (t) => {
+    //@ts-ignore
```

Comment:
These `@ts-ignore`s are pretty awkward.

What I usually do is force a type assertion:
```
 const plan = {
        options: {
            start: 'a',
        }
    } as ExecutionPlan;
```

This usually does the trick. Sometimes you need to hack it with `as unknown as ExecutionPlan`